src/vmax.rs

```diff
@@ -475,7 +474,7 @@ pub struct FeDirectorMetrics {
     pub miss_reqs: f64,
     pub read_miss_reqs: f64,
     pub write_miss_reqs: f64,
-    pub percent_read_reqs: f64,
+    pub percent_re4ad_reqs: f64,
```

Comment:
Is this a typo?

Reply:
Yes! Corrected it. Will be fixed in next push.